configdialog.cpp

```diff
@@ -805,17 +998,17 @@ LRESULT CALLBACK ScreenSaverImportExportPalettesDialog(HWND dialog, UINT message
 			delete exported_palettes;
 			return TRUE;
 		} case WM_COMMAND: {
-			wchar_t palettes_buffer[1 << 16] { L'\0' };
+			std::unique_ptr<wchar_t> palettes_buffer(new wchar_t[1 << 16](L'\0'));
```

Comment:
I didn't like the warning of having 128KB+ of data on the stack, so I tried turning this into a unique pointer.

Reply:
Because our memory management needs are simple, we are quite alright using raw pointers and managing them ourselves. The smart pointer here is maybe better in a vacuum, but the inconsistency with the rest of the code is confusing (why is _this_ a smart pointer? Why is it special?)